ui/src/message_popup/playtest/smithB_scenario.jsx

```diff
@@ -0,0 +1,405 @@
+  <br />
+  <div>Jamal: “Let’s send, ‘I am Groot.’”</div>
+  <br />
+  <div>Kevin: “Hahaha yeah! What a great movie. Have you seen the new one?”</div>
+  <br />
+  <div>Jamal: “No not yet. Really want to though.”</div>
+  <br />
+  <div>Kevin: “You have to. It’s just as good. Next, try sending it in bold.”</div>
+</div>,
+  write: true
+  });
+
+  slides.push({type: 'Try it!', el:
+<div>   
+  <div>Mr. Smith is holding group discussion at the end of class:</div>
+  <div><img src="https://s3-us-west-2.amazonaws.com/tsl-public/threeflows/smith-scenario/MA-SmithGroup2.jpg" style={{display: 'block', width: '90%', margin: 20}} /></div>
+  <br />
+  <div>Mr. Smith: “What are some benefits of the ASCII protocol?”</div>
+  <br />
+  <div>Jamal: “In only 128 possible characters, it allows you to send all the common characters and numbers.”</div>
+  <br />
+  <div>Mr. Smith: “Yeah, anyone else?”</div>
+  <br />
+  <div>Tim: “It includes both capital and lowercase letters, which is nice.”</div>
+  <br />
+  <div>Mr. Smith: “That’s great! It is cool isn’t it? Thanks for sharing, Tim.” </div>
+</div>,
+  write: true
+});
+
+  slides.push({type: 'Try it!', text:
+`
+That's the end of the class! Take a moment to reflect on how you felt the class went overall.
+` 
+
+});
+    
+
+  slides.push({type: 'Try it!', text:
+`Once the students have left, your friend Mr. Smith comes up to you and asks for your thoughts. What feedback would you give him about what you observed?
+ `,
+  write: true
+
+});
+    // ---------------------------------
+    // PAUSE 
+    // ---------------------------------
+
+  slides.push({type: 'PAUSE!', text:
+  `PAUSE HERE
+
+
+Pause here and return to the group. We will continue with this part of the simulation after a brief group discussion. 
+`});
+
+    // ---------------------------------
+    // Round TWO
+    // ---------------------------------
+
+  slides.push({type: 'Set Context - Lenses', el:
+    <div>
+    <div>Mr. Smith comes back to you and asks you to observe another class of his later in the day. It will cover similar subject matter. For this viewing, Mr. Smith has specifically asked you to focus on student engagement and equity. </div>
+    <br />
+    <div>Questions to ask yourself while viewing the scenes may be:</div>
+    <br />
+    <div>- Which students seem to be engaged with the activity?</div>
+    <div>- How are the interactions impacting the learning and confidence of the students involved?</div>
+    <br />
+    <div>Feel free to jot down notes for each slide and be prepared to give new feedback to Mr. Smith!</div>
+    </div>
+    });
+
+
+slides.push({type: 'Set Context - Lenses', text:
+`The new class being taught by Mr. Smith also has 6 students. The demographics of the class are as follows:
+
+  4 Male Students
+  2 Female Students
+
+  3 White Students (2 Male, 1 Female)
+  1 African American Student (Female)
+  1 Asian Student (Male)
+  1 Hispanic/Latino Student (Male)
```

Comment:
Just noticing this now (similarly for A), but this seems like a lot of data, particularly with slicing it by different dimensions.

Would a a list of each students' name, with their race, ethnicity and  gender be simpler?